src/main/java/learnositysdk/request/Init.java

```diff
@@ -424,9 +428,34 @@ private void setRequestPacket(Object requestPacket) throws Exception
     private void updateRequestString() {
         this.requestString = this.requestPacket.toString();
 
-        // Undo escaped forward slashes by JSONObject.toString to match PHP sdk
-        // https://github.com/Learnosity/learnosity-sdk-php/blob/42f2ffade51831966e0f92f25fa06e50c1dbecce/src/LearnositySdk/Utils/Json.php#L54
-        this.requestString = this.requestString.replace("\\/", "/");
+        this.requestString = fixJson(this.requestString);
+    }
+
+    /**
+     * Ensures encoded JSON is correctly formatted for the Learnosity APIs
+     *
+     * PHP SDK: https://github.com/Learnosity/learnosity-sdk-php/blob/42f2ffade51831966e0f92f25fa06e50c1dbecce/src/LearnositySdk/Utils/Json.php#L54
+     */
+    private String fixJson(String string) {
+        final Pattern pattern = Pattern.compile("\\\\u([0-9A-Fa-f]{4})");
```

Comment:
Reminding myself that this is a regex for catching unicode strings (like `\u2013`)